omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -103,7 +103,28 @@
               {actions_vhost, <<"<%= @actions_vhost %>">>},
               {actions_exchange, <<"<%= @actions_exchange %>">>},
               {actions_fqdn, <<"<%= node['private_chef']['lb']['api_fqdn'] %>">>},
-
```

Comment:
Can we move these into {rabbitmq, [{management, [ ...proplist ...]}]}? It's an extra step to extract it, but a bit cleaner. (I've been wanting to do the same with the actions_ prefixed stuff too, but that'll require a change to reporting since I think it uses this config block.)